public/docs/ts/latest/guide/npm-packages.jade

```diff
@@ -150,16 +150,16 @@ a(id="dev-dependencies")
   They do not have to be deployed with the production application although there is rarely harm in doing so.
 
   ***[concurrently](https://www.npmjs.com/package/concurrently)*** - 
-  a utility to run multiple *npm* commands concurrently on OS/X, Windows, and Linux operating systems.
+  A utility to run multiple *npm* commands concurrently on OS/X, Windows, and Linux operating systems.
```

Comment:
Thanks for taking a look at the PR so fast. I think "A" is correct in this case because of the phonetics involved. Other similar examples would be "A university", "A UI", "A user guide", etc. It looks like Oxford uses "A" as well: http://www.oxforddictionaries.com/us/definition/american_english/utility